packages/@react-spectrum/s2/src/ComboBox.tsx

```diff
@@ -304,6 +305,13 @@ const dividerStyle = style({
   width: 'full'
 });
 
+const avatar = style({
```

Comment:
The style macro variables in Picker.tsx and ComboBox.tsx are named a bit inconsistently, do you prefer this to be called `avatar` or `avatarStyle(s)`?